src/absil/il.fs

```diff
@@ -1558,7 +1558,7 @@ type ILFieldDef =
     member x.WithStatic(condition) = { x with Attributes = x.Attributes |> conditionalAdd condition FieldAttributes.Static }
     member x.WithSpecialName(condition) = { x with Attributes = x.Attributes |> conditionalAdd condition (FieldAttributes.SpecialName ||| FieldAttributes.RTSpecialName) }
     member x.WithNotSerialized(condition) = { x with Attributes = x.Attributes |> conditionalAdd condition FieldAttributes.NotSerialized }
-    member x.WithLiteral(condition) = { x with Attributes = x.Attributes |> conditionalAdd condition FieldAttributes.Literal }
+    member x.WithLiteralDefaultValue(literal) = { x with Attributes = x.Attributes |> conditionalAdd (isSome literal) FieldAttributes.Literal ||| FieldAttributes.HasDefault}
```

Comment:
@KevinRansom  shouldn't this also set the LiteralValue?

Reply:
@forki, it should, but I got excited about incorporating DefaultValue as well, anyway it doesn't even compile. So clearly more work is required.

Reply:
;-)